test/Microsoft.Framework.Runtime.Tests/VersionUtilityFacts.cs

```diff
@@ -57,14 +58,42 @@ public class VersionUtilityFacts
         [InlineData("dnxcore50", "portable-net45+win8", true)]
         [InlineData("dnxcore50", "portable-net451+win81", true)]
         [InlineData("dnxcore50", "portable-net40+sl5+win8", false)]
-        public void FrameworksAreCompatible(string projectTargetFramework, string packageTargetFramework, bool compatible)
+
+        // Core50
+        [InlineData("core50", "core50", true)]
```

Comment:
Unrelated to this pull request, but noticed from the test data:

Is `portable-net451+win81` compatible with `portable-net40+sl5+win8`? I would have thought so, since the two TFMs are present-and-equal-or-lower.

Given that, if `core50` is compatible with `portable-net451+win81` and not compatible with `portable-net40+sl5+win8` does that mean TFM compatibility is non-transitive? Is there even such a thing as a non-transitive compatibility scheme??